### PR TITLE
largest-series-product: add to track

### PR DESCRIPTION
**Pull request by stkent**

### CODE REVIEW

File: exercises/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+public final class LargestSeriesProductCalculatorTest {
+
+    @Rule
```
Comment (stkent) [inline exercises/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java]: Once discussion in #171 is complete, I will update with an appropriate comment to introduce the user to JUnit `Rule`s here.

Comment (jtigger) [inline exercises/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java]: The way you did it there was right on.

Comment (jtigger) [inline exercises/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java]: I'll hold off on merging until you signal you're done with this part.

Comment (stkent) [inline exercises/largest-series-product/src/test/java/LargestSeriesProductCalculatorTest.java]: Comment has now been added!

**Pull request merged**